week-2.md

```diff
@@ -0,0 +1,65 @@
+# Week Two Info
+
+> "The solution was script, which generalizes the problem so transacting parties can describe their transaction as a predicate that the node network evaluates.  The nodes only need to understand the transaction to the extent of evaluating whether the sender's conditions are met." -- Satoshi, [2010-06-17](https://bitcointalk.org/index.php?topic=195.msg1611#msg1611)
+
+Hopefully you've made it through the first week, successfully contacted
+your study group, caught up on any background about the process, and
+made some initial progress with review! If you haven't, that's fine --
+catch up as best you can, and feel free to ask questions about earlier
+topics even if we've supposedly moved on to later topics.
+
+Per the [curriculum](Curriculum.md) this week moves on from the overall
+taproot construction to looking at key path spends and the merkle
+construction that allows to have a multitude of scripts as an alternative
+to the direct key path spend.
+
+## Results from Week One
+
+We had the first two Q&A sessions, and have logs of the conversations captured: [week 1 Q&A 1](http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-05-19.00.log.html),
+[week 1 Q&A 2](http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-07-02.00.log.html). There's some interesting discussion in both of them, so worth a look if you've got time!
+
+There's already been a few suggested changes to the BIPs, including [PR#122](https://github.com/sipa/bips/pull/122), [PR#126](https://github.com/sipa/bips/pull/126/files), and [PR#128](https://github.com/sipa/bips/pull/128/files) -- they're all pretty short, so if you haven't contributed to a BIP or Bitcoin core before, they make for pretty easy examples of how to contribute!
+
```

Comment:
I'd say this is the most significant thing arising from the week, so we should include it? https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017443.html

Reply:
makes sense!